src/js/tech/tech.js

```diff
@@ -127,33 +129,12 @@ class Tech extends Component {
     this.duration_ = this.duration();
   }
 
-  bufferedPercent() {
-    let bufferedDuration = 0,
-        start, end;
-
-    if (!this.duration_) {
-      return 0;
-    }
-
-    let buffered = this.buffered();
-
-    if (!buffered || !buffered.length) {
-      buffered = createTimeRange(0,0);
-    }
-
-    for (var i=0; i<buffered.length; i++){
-      start = buffered.start(i);
-      end   = buffered.end(i);
-
-      // buffered end can be bigger than duration by a very small fraction
-      if (end > this.duration_) {
-        end = this.duration_;
-      }
-
-      bufferedDuration += end - start;
-    }
+  buffered() {
+    return null;
```

Comment:
This is never called outside the unit tests. null to me seems more appropriate because their is no buffer at all.